### PR TITLE
feat(ci): add end-to-end github actions tests for standard stack

### PR DESCRIPTION
**Pull request by acburdine**
these will start working when #1318 and #1319 are merged in

### CODE REVIEW

File: .github/workflows/e2e-test.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        name: 'Setting up required services'
+      - run: 'echo -e "127.0.0.1 cli-testing.ghost.org\n" | sudo tee -a /etc/hosts'
+        name: 'Updating hosts file'
+      - uses: actions/checkout@v2
+        with:
+          path: 'cli'
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: cd cli && yarn && sudo yarn link --link-folder /bin/
```
Comment (vikaspotluri123) [inline .github/workflows/e2e-test.yml:23]: Is there a reason we need to use link? What's wrong with running `./cli/bin/ghost ...`?

File: .github/workflows/e2e-test.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      - name: Setting up Ghost instance
+        run: |
+          mkdir ghost
```
Comment (vikaspotluri123) [inline .github/workflows/e2e-test.yml:26]: Shouldn't be necessary (implicit test)

**Review by vikaspotluri123 — APPROVED** [review]
Couple q's, but everything else looks good. Once this is done, I'll rebase my PR and do a minimal matrix (ubuntu 16, ubuntu 18, ubuntu 20, ubuntu 20 MySQL 8)

File: .github/workflows/e2e-test.yml (hunk 1 above)
Comment (acburdine) [reply .github/workflows/e2e-test.yml:23]: that could work too - wanted to try and make it as close to real-world as possible though

File: .github/workflows/e2e-test.yml (hunk 2 above)
Comment (acburdine) [reply .github/workflows/e2e-test.yml:26]: good point, will remove 👍

File: .github/workflows/e2e-test.yml (hunk 1 above)
Comment (vikaspotluri123) [reply .github/workflows/e2e-test.yml:23]: Gotcha. Main reason I was asking is that seems to be the only difference between the 3 OSes - if they follow the same template, matrixing will be super easy

Comment (acburdine) [reply .github/workflows/e2e-test.yml:23]: I'll test and see if /usr/bin works on ubuntu 16/18.

Comment (acburdine) [reply .github/workflows/e2e-test.yml]: yup looks like `/usr/bin/` works consistently across all three - updated to just use that

**Pull request merged**